src/ResourceManager/Insights/Commands.Insights/ManagementCmdletBase.cs

```diff
@@ -35,11 +35,10 @@ public abstract class ManagementCmdletBase : InsightsCmdletBase, IDisposable
         /// </summary>
         public IInsightsManagementClient InsightsManagementClient
         {
-            get
-            {
+            // The premise is that a command to establish a context (like Add-AzureRmAccount) has been called before this command in order to have a correct CurrentContext
+            get {
```

Comment:
Please make line break in the comment at around 100th character. It is extremely hard to read long comments on github for anyone.